astropy/io/ascii/tests/test_c_reader.py

```diff
@@ -42,24 +41,24 @@ def assert_table_equal(t1, t2, check_meta=False, rtol=1.0e-15, atol=1.0e-300):
     Test equality of all columns in a table, with stricter tolerances for
     float columns than the np.allclose default.
     """
-    assert_equal(len(t1), len(t2))
-    assert_equal(t1.colnames, t2.colnames)
+    npt.assert_equal(len(t1), len(t2))
```

Comment:
Here `npt.assert_equal()` is being used to check lengths of lists. Compare the tracebacks:
```
============================================== FAILURES ===============================================
__________________________________________ test_example_good __________________________________________

    def test_example_good():
>       assert len(list1) == len(list2)
E       assert 2 == 3
E        +  where 2 = len([1, 2])
E        +  and   3 = len([1, 2, 3])

example.py:8: AssertionError
__________________________________________ test_example_bad ___________________________________________

    def test_example_bad():
>       np.testing.assert_equal(len(list1), len(list2))
E       AssertionError: 
E       Items are not equal:
E        ACTUAL: 2
E        DESIRED: 3

example.py:11: AssertionError
```